File: examples/javascript/constructs/write_function.js
// Ported from examples/lumen/constructs/write_function.lm by scripts/port_examples.py; edit the Lumen original, not this file.
process.stdout.write("Hello");
process.stdout.write(" ");
process.stdout.write("World");
process.stdout.write("!");
console.log("");
process.stdout.write("Numbers: ");
let i = 1;
while (i <= 5) {
    process.stdout.write(String(i));
    if (i < 5) {
        process.stdout.write(", ");
    }
    i = i + 1;
}
console.log("");

File: examples/python/constructs/write_function.py
import sys
# Ported from examples/lumen/constructs/write_function.lm by scripts/port_examples.py; edit the Lumen original, not this file.
sys.stdout.write("Hello")
sys.stdout.write(" ")
sys.stdout.write("World")
sys.stdout.write("!")
print("")
sys.stdout.write("Numbers: ")
i = 1
while i <= 5:
    sys.stdout.write(str(i))
    if i < 5:
        sys.stdout.write(", ")
    i = i + 1
print("")
